src/components/organization/UnclassifiedGroupRow.tsx

```diff
@@ -0,0 +1,63 @@
+import { cva, VariantProps } from 'class-variance-authority';
+
+import EditBtn from './EditBtn';
+
+import { cn } from '@/utils/cn';
+
+const unclassifiedGroupRowVariants = cva(
+  'whitespace-nowrap w-full flex h-9 items-center py-2.5 pr-2 pl-3.5 gap-1 border-b border-solid border-borderGray text-sm font-medium  hover:bg-backgroundGray transition-all text-textGray2',
+  {
+    variants: {
+      state: {
+        default: 'bg-white ',
+        hold: 'bg-[rgba(25,240,120,0.2)]',
+        unselected: 'bg-backgroundUnselected',
+        selected: 'text-backgroundGreen bg-backgroundSelected',
+      },
+    },
+
+    defaultVariants: {
+      state: 'default',
+    },
+  }
+);
```

Comment:
```suggestion
const unclassifiedGroupRowVariants = cva(
  'whitespace-nowrap w-full flex h-9 items-center py-2.5 pr-2 pl-3.5 gap-1 border-b border-solid border-borderGray text-sm font-medium hover:bg-backgroundGray transition-all text-textGray2',
  {
    variants: {
      state: {
        default: 'bg-white',
        hold: 'bg-[rgba(25,240,120,0.2)]',
        unselected: 'bg-backgroundUnselected',
        selected: 'text-backgroundGreen bg-backgroundSelected',
      },
    },

    defaultVariants: {
      state: 'default',
    },
  }
);
```

🤔 아마 prettier-plugin-tailwindcss 가 className이라고 인식을 못 해서 포매팅을 자동으로 안 해 줬나 봐요

변수명을 `어쩌구저쩌구ClassName` 이런 식으로 지으면 인식하는 거 같더라구요

Reply:
뒤에 ClassName 붙어야 인식하는거네요 🤔 🤔  감사합니다